sites/fast-component-explorer/app/web-components/index.tsx

```diff
@@ -26,6 +35,59 @@ interface RenderDevToolsTabsConfig {
     schemaTabPanelContent: string;
 }
 
+function renderScenarioFastOptions(scenarioOptions: Array<Scenario>): React.ReactNode {
```

Comment:
I think this function can just be called `renderScenarioOptions`.

Reply:
Done